File: TUF/Array/RearrangeArray.ts
function rearrangeArray(arr: number[]): void {
    arr.sort((a, b) => a - b);
    let result: number[] = [];
    for (let i: number = 0; i < arr.length / 2; i++) {
        result.push(arr[i]);
    }
    for (let i: number = arr.length - 1; i >= arr.length / 2; i--) {
        result.push(arr[i]);
    }
    console.log(result);
}

let arr: number[] = [5, 4, 7, 8, 3, 2, 9];
rearrangeArray(arr);

export { };

File: TUF/Array/RearrangeArray.java
import java.util.Arrays;
import java.util.Scanner;

public class RearrangeArray {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int size = input.nextInt();
        int[] arr = new int[size];
        for (int i = 0; i < size; i++) {
            System.out.printf("Enter %d number element of the array: ", i + 1);
            arr[i] = input.nextInt();
        }
        input.close();
        rearrangeArray(arr);
    }

    private static void rearrangeArray(int[] arr) {
        Arrays.sort(arr);
        for (int i = 0; i < arr.length / 2; i++) {
            System.out.println(arr[i] + " ");
        }
        for (int i = arr.length - 1; i >= arr.length / 2; i--) {
            System.out.println(arr[i] + " ");
        }
    }
}
